source/hardware/controllers/behringer_bcd2000.rst

```diff
@@ -1,10 +1,11 @@
 .. _behringer-bcd2000:
 
-Behringer B-Control Deejay BCD 2000
-===================================
+Behringer BCD2000
+=================
```

Comment:
I'm unsure if its a good idea to shorten the name. On their website, Behringer refers to their products by their full name: https://www.behringer.com/product.html?modelCode=P0234

Reply:
Mhmm, makes sense. Then keep using the short version for headings and so forth.

Reply:
my initial impulse to rename was just for increasing convenience and quick lookup, because I couldn't spot the DDM4000 right away because it was named "Behringer Digital Pro Mixer DDM4000" or something.
that "issue" is resolved now, but since DDM, BCR, ... are just acronyms  we may very well include the expanded names in the first paragraph.

Reply:
I added the expanded names as requested.